### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by pankaj-prog**

### CODE REVIEW

File: src/pages/authentication/auth.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+.auth-form {
+  padding: 1.5rem;
+  border-top: 1px solid var(--LIGHT-GRAY);
+  display: flex;
+  width: fit-content;
+  max-width: 42ch;
+  flex-direction: column;
+  gap: 0.5rem;
+}
```
Comment (manojsarna) [inline src/pages/authentication/auth.css:26]: can use px values!!

File: index.html [hunk 2]
```diff
@@ -15,7 +15,9 @@
           <label for="search"><i class="fas fa-search"></i></label>
           <input type="text" id="search" />
         </div>
-        <button class="btn btn-solid-primary btn-rc">Log in to play</button>
+        <button class="btn btn-solid-primary btn-rc">
+          <a href="/src/pages/authentication/login.html">Log in to play</a>
+        </button>
```
Comment (bhtibrewal) [inline index.html:20]: change: place the anchor tag over the button

File: src/pages/authentication/signup.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          <p class="text-muted text-sm form-alert">
+            By signing up, you agree to our Terms of Use and Privacy Policy.
+          </p>
+        </form>
+        <footer class="auth-box-footer">
+          Already have an account ?
+          <button class="btn btn-link-primary">
+            <a href="/src/pages/authentication/login.html"> Log in</a>
+          </button>
+        </footer>
```
Comment (bhtibrewal) [inline src/pages/authentication/signup.html:71]: Suggestion: i dont think you should use footer in the login section, this whole should be in the form itself

File: src/pages/authentication/auth.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  padding: 1.5rem;
+}
+
+.auth-form {
+  padding: 1.5rem;
+  border-top: 1px solid var(--LIGHT-GRAY);
+  display: flex;
+  width: fit-content;
+  max-width: 42ch;
+  flex-direction: column;
```
Comment (manojsarna) [inline src/pages/authentication/auth.css:24]: ch is characters , right?

File: src/pages/authentication/signup.html [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+          <label for="search"><i class="fas fa-search"></i></label>
+          <input type="text" id="search" />
+        </div>
+        <button class="btn btn-solid-primary btn-rc">
+          <a href="/src/pages/authentication/login.html">Log in</a>
+        </button>
+      </header>
+    </section>
+    <section class="site-main">
+      <section class="auth-box">
```
Comment (bhtibrewal) [inline src/pages/authentication/signup.html:24]: Change : section inside section , replace this with div

File: src/pages/authentication/login.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+          <input type="text" id="search" />
+        </div>
+        <button class="btn btn-solid-primary btn-rc">
+          <a href="/src/pages/authentication/signup.html">Sign up</a>
+        </button>
+      </header>
+    </section>
+    <section class="site-main">
+      <section class="auth-box">
+        <h1 class="auth-box-heading h4">Log in to your account</h1>
```
Comment (bhtibrewal) [inline src/pages/authentication/login.html:25]: why use h4 class with h1 tag?  please change this

File: src/pages/authentication/auth.css [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  cursor: pointer;
+  transition: background-color 0.5s ease;
+}
+
+.social-btn {
+  background-color: var(--OFFWHITE);
+  border: 1px solid var(--GRAY);
+  display: flex;
+  gap: 8px;
+  align-items: center;
```
Comment (manojsarna) [inline src/pages/authentication/auth.css:41]: be consistent with px values or rems

File: src/pages/authentication/auth.css [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+.fa-google {
+  background: linear-gradient(
+    171deg,
+    rgba(255, 0, 0, 1) 0%,
+    rgba(251, 0, 0, 1) 23%,
+    rgba(248, 249, 0, 1) 40%,
+    rgba(16, 0, 255, 1) 54%,
+    rgba(0, 14, 255, 1) 100%
+  );
+
```
Comment (manojsarna) [inline src/pages/authentication/auth.css:72]: check for color vars

File: src/pages/authentication/auth.css [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  color: blue;
+}
+
+.input-container {
+  font-size: 1rem;
+  border: 1px solid var(--BLACK);
+  padding: 0.8rem;
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
```
Comment (manojsarna) [inline src/pages/authentication/auth.css:87]: check here for px values

**Review by manojsarna — APPROVED** [review]
overall codes looks great

do check the comments!!

File: src/pages/authentication/auth.css (hunk 7 above)
Comment (pankaj-prog) [reply src/pages/authentication/auth.css:41]: okay

**Review by bhtibrewal — APPROVED** [review]
I have written a few changes, else looks good to merge

File: index.html (hunk 2 above)
Comment (pankaj-prog) [reply index.html:20]: okay

File: src/pages/authentication/signup.html (hunk 5 above)
Comment (pankaj-prog) [reply src/pages/authentication/signup.html:24]: okay

File: src/pages/authentication/login.html (hunk 6 above)
Comment (pankaj-prog) [reply src/pages/authentication/login.html:25]: h4 is for styling

File: src/pages/authentication/signup.html (hunk 3 above)
Comment (pankaj-prog) [reply src/pages/authentication/signup.html:71]: it is intended to be in section auth box instead of form

**Pull request merged**